packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -43,6 +43,51 @@ const Page = (): ReactElement => {
         export default App;
         `}
       </Sandbox>
+      <Title>Adding icons</Title>
+      <Description>
+        Please validate in the all icons story below in case the icon you wish to add is already
+        added. Steps for adding a new icon from Figma:
+      </Description>
+      <ul>
+        <li>
+          <Description>
+            Visit the [icons figma
+            file](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=57%3A0&t=xzz6HfFC50U8iuJp-0)
+          </Description>
+        </li>
+        <li>
+          <Description>
+            Click the icon you wish to add on Figma. Select the root icon element and not the layer
+            within. Copy this root icon as SVG. **Note**: generally the root svgs have a viewbox of
+            `0 0 24 24`.
+          </Description>
+        </li>
+        <li>
+          <Description>
+            ![](https://user-images.githubusercontent.com/24487274/203279145-1e0b0540-467d-4901-97c8-014f98b3cfca.png)
```

Comment:
Can we remove the bullet point to the left of this image?

Reply:
I realized we can't use `li` and `ul` because this also needs to work on react native. Did some updates in https://github.com/razorpay/blade/pull/883/commits/6bd812011991d0077ccfdbc28db1625c68607145